index.html

```
@@ -1,237 +1,189 @@
-					<picture>
-						<source srcset="img/index_photos/webp/10.webp" type="image/webp">
-						<img src="img/index_photos/10.jpg">
-					</picture>
-					<picture>
-						<source srcset="img/index_photos/webp/11.webp" type="image/webp">
-						<img src="img/index_photos/11.jpg">
-					</picture>
+		<main class="main">
+			<div class="hero">
+				<div class="slide-track">
+					<img class="slide" src="/img/index_photos/2.jpg" data-header-color="black">
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)